modules/data-entry/src/main/frontend/src/dataHomepage/NewFormDialog.jsx

```diff
@@ -39,16 +39,14 @@ import FormattedText from "../components/FormattedText.jsx";
 
 const PROGRESS_SELECT_QUESTIONNAIRE = 0;
 const PROGRESS_SELECT_SUBJECT = 1;
-export const MODE_ACTION = 0;
-export const MODE_DIALOG = 1;
 
 /**
- * A component that renders a FAB to open a dialog to create a new form.
+ * A component that renders a dialog to create a new form, and optionally a FAB to activate the dialog.
  *
  * @param {presetPath} string The questionnaire to use automatically, if any.
  */
 function NewFormDialog(props) {
-  const { children, classes, presetPath, currentSubject, theme, mode, open, onClose } = { mode: MODE_ACTION, open: false, ...props };
+  const { classes, presetPath, currentSubject, theme, mode, open, onClose, withButton, buttonTitle } = {open: false, ...props };
```

Comment:
Forgot to remove this (now unused) property:

```suggestion
  const { classes, presetPath, currentSubject, theme, open, onClose, withButton, buttonTitle } = {open: false, ...props };
```